### DISCUSSION TITLE
Adding Alien Archive creatures?

### DISCUSSION THREAD

**Discussion by magee101** [wildj79/foundryvtt-starfinder#1210] [General]
Hello and sorry if this isn't quite the right place. My intentions today was to ask for a specific monster to be added from Starfinder Alien Archive book; page 94

Patrol-Class Security Robot

The lower CR version Observer-Class and the higher CR version Terminator-Class is available in the compendium but the Patrol-Class is not.

Thanks for reading! -Magee

**Comment by ian612** [comment]
Hi Magee, for missing compendium creatures, starships, items, etc. you can make an "Issues" post and tag it with the "compendium" tag. If you do that it'll get noticed by the data entry team quickly. Once things are noted as missing they're generally very quick to add them in.
